File: linearsearch.js
function linearsearch(A, x, lo, hi) {
	for (let i = 0; i < A.length; i++) {
		if (A[i] == x) {
			return i;
		}
	}
	return -1;
}

mylist = [ 1, 2, 3, 4, 5, 6, 7, 8, 8 ];

let result = linearsearch(mylist, 5, 0, 99);

console.log('The result is ' + result);

File: LinearSearch.java
public class LinearSearch {
   public static void main(String[] args) {
      int[] A = new int[10];
      for (int i = 0; i < 10; i++) {
         A[i] = i * i;
      }
      int result = search(A, 8, 0, 10);
      System.out.println(result);
   }

   public static int search(int[] A, int x, int lo, int hi) {
      for (int i = lo; i < hi; i++) {
         if (A[i] == x) {
            return i;
         }
      }
      return -1;
   }
}
